src/qml/CMakeLists.txt

```diff
@@ -34,3 +34,30 @@ set_target_properties(qfield_qml PROPERTIES LINKER_LANGUAGE CXX)
 
 # For QtCreator
 set(QML_IMPORT_PATH "${CMAKE_CURRENT_SOURCE_DIR}/imports")
+
+file(GLOB qfield_qmlfiles "${CMAKE_SOURCE_DIR}/src/qml/*.qml")
+file(GLOB qfield_qmlfiles_geometryeditors
+     "${CMAKE_SOURCE_DIR}/src/qml/geometry_editors/*.qml")
+file(GLOB qfield_qmlfiles_editorwidgets
+     "${CMAKE_SOURCE_DIR}/src/qml/editorwidgets/*.qml")
+file(GLOB qfield_qmlfiles_theme
+     "${CMAKE_SOURCE_DIR}/src/qml/imports/Theme/*.qml")
+if(BUILD_WITH_QT6)
+  file(GLOB qfield_qmlfiles_theme_extra
+       "${CMAKE_SOURCE_DIR}/src/qml/imports/Theme/+Qt6/*.qml")
+  file(GLOB qfield_qmlfiles_qfieldcontrols
+       "${CMAKE_SOURCE_DIR}/src/qml/imports/QFieldControls/+Qt6/*.qml")
+else()
+  file(GLOB qfield_qmlfiles_theme_extra
+       "${CMAKE_SOURCE_DIR}/src/qml/imports/Theme/+Qt5/*.qml")
+  file(GLOB qfield_qmlfiles_qfieldcontrols
+       "${CMAKE_SOURCE_DIR}/src/qml/imports/QFieldControls/+Qt5/*.qml")
+endif()
+list(
+  APPEND
+  qfield_qmlfiles
```

Comment:
I think a `set(...PARENT_SCOPE...)` will be needed to make this var accessible from elsewhere.

Reply:
@m-kuhn , TIL :bowing_man:

Reply:
@m-kuhn , done.